### DISCUSSION TITLE
Chia not syncing in windows 10 pro

### DISCUSSION THREAD

**Discussion by androidmasri** [Chia-Network/chia-blockchain#17018] [Support]
Chia is not syncing in my windows 10 pro machine, it stopped syncing since 24 November 2023, please advice, kindly find attached screenshot.
Regards,
Mohammed
Email: android.masri@gmail.com
![unnamed (1)](https://github.com/Chia-Network/chia-blockchain/assets/115132815/2e91519b-2152-4e5e-bfe9-1ca5f114d787)

**Comment by BrandtH22** [comment]
Hey @androidmasri , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

I would first ensure that your system time is correct, I personally use this site to do so: https://time.is/

Are you seeing any errors in your logs?
Once you have synced to height are you still connected to peers and downloading data from them?

A good place to start will be this doc: https://docs.chia.net/check-if-things-are-working/

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
